.gitignore

```diff
@@ -3,3 +3,4 @@ composer.lock
 phpstan.neon
 phpunit.xml
 vendor
+.idea
```

Comment:
Please don't add this. Put this in your global ignore file.

Reply:
cant see the checkbox for "editing by maintainers"